packages/video_player_avplay/tizen/src/plus_player.cc

```diff
@@ -139,6 +139,10 @@ int64_t PlusPlayer::Create(const std::string &uri,
 
   int64_t start_position = flutter_common::GetValue(
       create_message.player_options(), "startPosition", (int64_t)0);
+  if (start_position == 0) {
+    start_position = flutter_common::GetValue(create_message.player_options(),
+                                              "startPosition", (int32_t)0);
```

Comment:
I was looking for a way to use std::variant<int32_t, int64_t> and std::holds_alternative<T>, std::get<int32_t>.
But I gave up because it seemed like it would only increase code complexity.

But I think it's a code that should be improved someday. Could you please leave a comment in code(//TODO?) why int32_t is needed?

Reply:
I also realized this problem, we can release it for 3rd-party first. I will implement a new function for all types later.